Triadic Census for Undirected Graph

Is there any specific reason (for e.g. computationally expensive, etc.) why the [triads algorithms](https://networkx.org/documentation/stable/reference/algorithms/triads.html) are not implemented for Undirected Graphs ? From what I understand, there would be only 4 types of triads possible in an undirected graph and the calculation would be similar to how it's done for directed graph. Please let me know if I am missing something here. Also, can I please go ahead with the implementation for the triad algorithms ?  :)

> go for it. :)
@dschult I updated the documentation as per the issue and made a pull request. All checks passed except "reviewer label". I am new to this, but I think there is a need for a reviewer? 

The situation is: 
Run echo "This PR needs a reviewer to label the PR with a 'type:' label"
  echo "This PR needs a reviewer to label the PR with a 'type:' label"
  exit 1
  shell: /usr/bin/bash -e {0}
  env:
    LABELS: 
This PR needs a reviewer to label the PR with a 'type:' label
Error: Process completed with exit code 1.

Thank you for availabilty!

--- a/networkx/algorithms/triads.py
+++ b/networkx/algorithms/triads.py
@@ -178,6 +178,10 @@ def triadic_census(G, nodelist=None):
     This algorithm has complexity $O(m)$ where $m$ is the number of edges in
     the graph.
 
+    For undirected graphs, the triadic census can be computed by first converting
+    the graph into a directed graph using the ``G.to_directed()`` method. Afther this conversion,
+    only the triad types 003, 102, 201 and 300 will be present in the undirected scenario.
+
     Raises
     ------
     ValueError

--- a/networkx/algorithms/triads.py
+++ b/networkx/algorithms/triads.py
@@ -180,7 +180,7 @@ def triadic_census(G, nodelist=None):
 
     For undirected graphs, the triadic census can be computed by first converting
     the graph into a directed graph using the ``G.to_directed()`` method.
-    After this conversion, only the triad types 003, 102, 201 and 300 will be 
+    After this conversion, only the triad types 003, 102, 201 and 300 will be
     present in the undirected scenario.
 
     Raises

--- a/networkx/algorithms/triads.py
+++ b/networkx/algorithms/triads.py
@@ -179,8 +179,9 @@ def triadic_census(G, nodelist=None):
     the graph.
 
     For undirected graphs, the triadic census can be computed by first converting
-    the graph into a directed graph using the ``G.to_directed()`` method. Afther this conversion,
-    only the triad types 003, 102, 201 and 300 will be present in the undirected scenario.
+    the graph into a directed graph using the ``G.to_directed()`` method.
+    After this conversion, only the triad types 003, 102, 201 and 300 will be 
+    present in the undirected scenario.
 
     Raises
     ------

